Training6/training-6-sagas-login/src/components/Login.js

```diff
@@ -0,0 +1,92 @@
+import { Button, Form, Input } from "antd";
+import React from "react";
+import { connect } from "react-redux";
+import { Redirect } from "react-router-dom";
+import { onLoginRequest } from "../actions/index";
+import "antd/dist/antd.css";
+import "./login.css";
+
+function Login({ onLoginRequest, login }) {
+  const layout = {
+    labelCol: {
+      span: 8
+    },
+    wrapperCol: {
+      span: 16
+    }
+  };
+  const tailLayout = {
+    wrapperCol: {
+      offset: 8,
+      span: 16
+    }
+  };
+  console.log(localStorage.getItem("token"));
+  const onFinish = values => {
+    onLoginRequest(values);
+  };
+
+  const onFinishFailed = errorInfo => {
+    console.log("Failed:", errorInfo);
+  };
+
+  if (localStorage.getItem("token")) {
```

Comment:
Please try to using redux-persist to persist your token @nguyentainguyen 
https://github.com/rt2zz/redux-persist